tests/test_hidpp20.py

```diff
@@ -0,0 +1,41 @@
+import os
+import pytest
+
+from . import TestBase
+
+
+class TestHIDPP20(TestBase):
+    shortname = 'logitech-g-pro-wireless'
```

Comment:
Hmm, this won't work for features that are not available on a G Pro Wireless device.

One solution is to have the TestHIDPP20 class hidden, and then add a subclass GPro, which has all of the feature tests matching the G Pro capability. This won't solve all of teh problems, but at least it will help us extend the tests easier

Reply:
I was thinking of have a test device with all the features.

Reply:
then the device will not be a consumer one, which is not exactly what we asked for.

Reply:
Well, we can have a base test class that implements all the features and then subclass it and enable the tests according to the device feature table.

> then the device will not be a consumer one, which is not exactly what we asked for.

We are just testing the protocol, so we will end up just running the same tests again and again, but sure, I can implement it :)

Reply:
This is out of scope for this PR, I added a `TODO` in the code.